app/models/message.rb

```diff
@@ -26,6 +26,7 @@ class Message < ApplicationRecord
   validates :body, presence: true
 
   after_create_commit :broadcast_message
+  after_create_commit :notify_user
```

Comment:
When possible, avoid adding callbacks to models. Instead, try extracting the logic into the service or class from where the message is created. Just for your information.